src/pudl/metadata/resources/__init__.py

```diff
@@ -289,8 +295,15 @@
             "data_maturity",
         ],
         "map_cols_dict": {
-            "operator_utility_id_eia": "utility_id_eia",
-            "operator_name": "utility_name_eia",
+            "owner_utility_id_eia": "utility_id_eia",
+            "owner_name": "utility_name_eia",
+            "owner_state": "state",
+            "owner_country": "country",
+            "owner_street_address": "street_address",
+            "owner_zip_code": "zip_code",
+            "owner_city": "city"
+            # currently can't harvest operator utilities in the same dict
```

Comment:
Ideally you'd be able to harvest the operator utilities from the ownership table as well, but as this currently functions, `map_cols_dict` would rename both the `operator_utility_id_eia` and `owner_utility_id_eia` columns to `utility_id_eia` and have colliding column names. Maybe it's fine to just not harvest the operator utilities from the ownership table.

Reply:
@cmgosnell did we still want to try adding a stripped down 2nd version of the ownership table with just the operator utility IDs into the harvesting for utilities?

Reply:
How do I run just the foreign key checks on the full DB? I thought they were part of the CI because I had one fail for `plant_parts_eia` in GitHub.

Reply:
Nevermind figured it out

Reply:
I haven't generated the FERC side but the `glue` tables and EIA tables passed the foreign key checks.

Reply:
i think if you wanted to harvest both `map_cols_dict` would need to be a list of map dicts that you'd add both renamed versions into `dfs` -> `compiled_df`